tree/ntuple/v7/src/RNTupleModel.cxx

```diff
@@ -416,17 +423,14 @@ std::unique_ptr<ROOT::Experimental::REntry> ROOT::Experimental::RNTupleModel::Cr
 
 ROOT::Experimental::REntry::RFieldToken ROOT::Experimental::RNTupleModel::GetToken(std::string_view fieldName) const
 {
-   if (!IsFrozen())
-      throw RException(R__FAIL("invalid attempt to get field token of unfrozen model"));
-
```

Comment:
This seems to be a second, strictly speaking unrelated change that allows creating tokens for models under construction. This is fine but we may want to call it out in the PR description.

Reply:
Correct, I ran into this for the test of field tokens with parallel writing and decided to include it in this PR (instead of freezing manually). I extended the description accordingly.